pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -390,6 +391,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 						},
 						Labels: map[string]string{
 							severityAlertLabelKey: "critical",
+							"runbook_url":         runbookUrlBasePath + "VirtApiRESTErrorsBurst",
```

Comment:
@assafad the "runbook_url" should be under Annotations

Reply:
Thanks, done